assets/prototype/domain/eventEditor/datetimes/DatesList/AddNewDateModal.tsx

```diff
@@ -1,12 +1,17 @@
+import React from 'react';
 import DateForm from '../dateForm/DateForm';
 import FormModal from '../../../../application/ui/components/forms/FormModal';
 import { useEntityMutator, EntityType } from '../../../../application/services/apollo/mutations';
 import useTickets from '../../data/queries/tickets/useTickets';
+import { AddItemModalProps } from '../../types';
+import { DateItemFormProps } from '../types';
 
-const AddNewDateModal = ({ onClose, isOpen }) => {
+const AddNewDateModal: React.FC<AddItemModalProps> = ({ onClose, isOpen }): JSX.Element => {
```

Comment:
DUDE!!! thought you said in chat you preferred `FunctionComponent ` !!! 😆 

I'm ok with using `FC` as long as we do it consistently

Reply:
VS Code was lazy to suggest and I was lazier to type it 😄